index.html

```diff
@@ -0,0 +1,234 @@
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="intro">
+      <div class="header__content">
+        <div class="heading">
+          <p class="heading-sub">
+            "Lets Connect and Build Africa."
+          </p>
+          <h1 class="heading-primary">
+            AFRICA YOUTH SUMMIT 2022
+          </h1>
+          <span class="heading-theme">
+            Equipping Africa's Youth for Continental Development.
+          </span>
+          <span class="heading-date">06.06.2022(MON) ~ 09(THUR)</span>
+          <span class="heading-venue">
+            @Kololo memorial independence grounds, Kampala Uganda
+          </span>
+        </div>
+      </div>
+    </section>
+
+    <section class="program-section" id="program">
+      <h2 class="section-headings">Main Program</h2>
+      <hr>
+      <ul class="program-section__events">
+        <li class="program-section__events--item">
+          <small class="program-section__events-icon">
+            <i class="fa fa-graduation-cap"><img src="images/education.png" alt="confenece"></i>
+          </small>
+          <h4 class="program-section__events-title">Lecture</h4>
+          <p class="program-section__events-info">
+            Listen to speakers from various African countries and backgrounds on Africa's Development.
+          </p>
+        </li>
+        <li class="program-section__events--item">
+          <small class="program-section__events-icon">
+            <i class="fa fa-cubes"><img src="images/conference.png" alt="education"></i>
+          </small>
+          <h4 class="program-section__events-title">Forum</h4>
+          <p class="program-section__events-info">
+            Appreciate various creations applying the African licence of Arts organised by Art center. 
+          </p>
+        </li>
+        <li class="program-section__events--item">
+          <small class="program-section__events-icon">
+            <i class="fa fa-desktop"><img src="images/conference.png" alt="confenece"></i>
+          </small>
+          <h4 class="program-section__events-title">Exhibition</h4>
+          <p class="program-section__events-info">
+            Have time to share your thoughts are experinces with experts for each topic.
+          </p>
+        </li>
+        <li class="program-section__events--item">
+          <small class="program-section__events-icon">
+            <i class="fa fa-bullseye"><img src="images/conference.png" alt="confenece"></i>
+          </small>
+          <h4 class="program-section__events-title">Workshop</h4>
+          <p class="program-section__events-info">
+            Try creating your own work and build on it other than
+            just watching art
+          </p>
+        </li>
+        <li class="program-section__events--item">
+          <small class="program-section__events-icon">
+            <i class="fa fa-cubes"><img src="images/education.png" alt="confenece"></i>
```

Comment:
currently your page is not responsive and breaks in the mobile version please fix this to improve the user experience